pyanaconda/installation.py

```diff
@@ -95,7 +97,26 @@ def doConfiguration(storage, payload, ksdata):
     os_config = TaskQueue("Installed system configuration", N_("Configuring installed system"))
     os_config.append(Task("Configure authselect", ksdata.authselect.execute))
     os_config.append(Task("Configure SELinux", ksdata.selinux.execute))
-    os_config.append(Task("Configure first boot tasks", ksdata.firstboot.execute))
+
+    services_proxy = SERVICES.get_proxy()
+    services_dbus_tasks = services_proxy.InstallWithTasks(util.getSysroot())
+    # add one Task instance per DBUS task
+    for dbus_task in services_dbus_tasks:
+        task_proxy = SERVICES.get_proxy(dbus_task)
+        os_config.append(Task(task_proxy.Name, sync_run_task, (task_proxy,)))
```

Comment:
There should be a method that will get and append all installation tasks of the given DBus service. This code is the same as for the Users module.

Reply:
Yeah, it looks like we will be using these task queues for now, so some tooling like this could be useful.